ligo/stablecoin/actions.ligo

```diff
@@ -336,8 +336,9 @@ function is_operator_action
   { const operator_transfer_policy : operator_transfer_policy_ =
       Layout.convert_from_right_comb ((store.permissions.operator : operator_transfer_policy))
   ; case operator_transfer_policy of
-      Operator_transfer_permitted (u) -> skip
-    | Operator_transfer_denied (u) -> failwith ("TX_DENIED")
+      No_transfer -> failwith ("TX_DENIED")
```

Comment:
Why do we have to make this check. Aren't we hard coding the behavior?

Reply:
I suppose this PR just needs to be rebased.